### DISCUSSION TITLE
도커 컨테이너 내 환경 설정 에러

### DISCUSSION THREAD

**Discussion by shinhakyum** [artjow/AI-ART#5] [Q&A]
pip install albumentations==0.4.3 opencv-python pudb==2019.2 imageio==2.9.0 imageio-ffmpeg==0.4.2 pytorch-lightning==1.4.2 omegaconf==2.1.1 test-tube>=0.7.5 streamlit>=0.73.1 einops==0.3.0 torch-fidelity==0.3.0 **git+https://github.com/openai/CLIP.git** protobuf~=3.20.1 torchmetrics==0.6.0 transformers==4.19.2 kornia==0.5.8 && pip uninstall -y torchtext
컨테이너 생성 후 VScode를 통해 접속한 후 gligen github의 환경설정을 참고하여 위 명령어를 실행하였는데  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/openai/CLIP.git /tmp/pip-req-build-8rlx1pkz
ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH? 다음과 같은 오류가 발생하였습니다 혹시 어떻게 해결해야하나요..?
![스크린샷 2023-12-27 195155](https://github.com/artjow/AI-ART/assets/149232501/56add594-ac7f-4443-9a0c-9c43af0dfa64)

**Comment by artjow** [comment]
VC에 git을 설치하세요.
터미널에서 아래 명령실행하세요.
1.Git 설치: 터미널에서 아래의 명령어를 사용하여 'git'을 설치합니다.
apt-get update
apt-get install git

2.경로 확인: 설치가 완료된 후, 'git' 명령어의 경로를 확인하여 시스템의 PATH에 추가되었는지 확인합니다.
which git

환경 변수 추가: 만약 'git'의 경로가 PATH에 없다면, 해당 경로를 PATH 환경 변수에 추가합니다. 예를 들어, 'git'이 '/usr/bin/git'에 설치되어 있다면, 다음과 같이 PATH에 추가할 수 있습니다.
export PATH=$PATH:/usr/bin/git

VSCode 재시작: 환경 변수 변경 사항을 적용하기 위해 VSCode를 재시작합니다.

명령어 재실행: 이제 'git' 관련 오류가 해결되었으므로, 처음에 실행하려고 했던 명령어를 다시 실행합니다.
